libs/postgres_backend/src/lib.rs

```diff
@@ -849,10 +849,6 @@ impl<IO: AsyncRead + AsyncWrite + Unpin> PostgresBackend<IO> {
             }
```

Comment:
You probably need to update comment for `handle_copy_stream_end()`.
Now it always closes the connection.

Reply:
done